app/Http/Controllers/PaystackWebhookController.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
i thinks your other pr blead into this one , we dont need this file in this piece of work